docs/user/ovirt/install_ipi.md

```diff
@@ -1,77 +1,132 @@
-   
-   One way to check if an IP is in use is to check if it has ARP associated with it 
-   - perform this check while on one of the hosts that would run the VMs:
-   ```console
-   $ arp 10.35.1.19
-   10.35.1.1 (10.35.1.1) -- no entry
-   ``` 
-2. RHCOS openstack image uploaded and ready as a template in oVirt
-    The `openshift-installer` has an hardcoded reference to its relevant supported images.
-    This information can be extracted easily and be used to download the qcow images
-    and upload it into ovirt-engine.
-    All of this is combined in that [ansible playbook](https://gist.github.com/rgolangh/adccf6d6b5eaecaebe0b0aeba9d3331b).  
-    >Note: that is a temporary solution and the installer will probably upload it
-    using the ovirt terraform provider.
-
-    For upstream users the FCOS image can be used from the built-in glance
-    provider - see this ansible role https://github.com/oVirt/ovirt-ansible-image-template
-    Use it to make a template from the glance image.
-
-## Install
-Start the installation by creating an `install-config` interactively, using a work-dir:
+
+1. Allocate 3 IP on the VM network: 
+    - IP for the internal kubernetes api, that all components will interact with 
+    - IP for the internal DNS service, to bootstrap etcd and to resolve names like 
+    `api.$CLUSTER_NAME.$CLUSTER_DOMAIN` and node names 
+    - IP for the Ingress, the load balancer in front of the cluster apps 
+    To work with this provider one must supply 2 IPs that are related to any MAC 
+    in the virtualization env, where the cluster will run. Those IPs will be active 
+    by keepalived, on, initially the bootstrap machine, and then the masters, after 
+    a fail-over, when the bootstrap is killed. 
+    Locate those IP's in the target network. If you want the network details, go to 
+    oVirt's webadmin and look for the designated cluster details and its networks. 
+    One way to check if an IP is in use is to check if it has ARP associated with it 
+    perform this check while on one of the hosts that would run the VMs: 
+       ```console
+       $ arp 10.35.1.19
+       10.35.1.1 (10.35.1.1) -- no entry
+       ```
+2. Name resolution of `API_VIP` from your installing machine 
+The installer must resolve the `API_VIP` during the installation, as it will 
+interact with the API to follow the cluster version progress. 
+To make it work prepend the DNS_VIP to your `/etc/resolv.conf` 
+    ```
+    search example.org
+    nameserver $DNS_VIP
+    ```
+
+
+## Minimum resources
+
+The default master/worker:
+- 4 CPUs
+- 16 RAM
+- 120 GB disk
+
+For 3 masters/3 workers, the target oVirt cluster must have at least:
+- 96RAM
+- 24vCPUs
+- 720GiB storage
+- 50IOPS for the Master VM disks, per ETCD requirement doc https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware.md
+
+> Worker count can be reduced to 2 in `install-config.yaml` in case needed.
```

Comment:
do we have any link to add as reference for install-config.yaml?

Reply:
It's a generated file. But I'll add a link to the installer explain command that will print information about the install-config.